src/main/webapp/WEB-INF/tags/admin/activity/activityLogTable.tag

```diff
@@ -20,8 +21,52 @@
                         <td colspan='2'><i>No application logs found</i></td>
                     </tr>
                 </c:if>
-                <c:forEach items="${logs}" var="log">
-                    ${log.logInfoForTableRowAsHtml}
+                <c:forEach items="${logs}" var="log" varStatus="count">
+                    <tr id="${count.first ? "first-row" : ""}">
+                        <td class="${log.tableCellClass}">
+                            <a onclick="submitLocalTimeAjaxRequest('${log.logTime}','${log.userGoogleId}','${log.displayedRole}', this);">
+                                ${log.displayedLogTime}
+                            </a>
+                            <p class="localTime"></p>
+                            <p class="${log.timeTakenClass}">
+                                <strong>${log.displayedLogTimeTaken}</strong>
+                            </p>
+                        </td>
+                        <td class="${log.tableCellClass}">
+                            <form method="get" action="${Const.ActionURIs.ADMIN_ACTIVITY_LOG_PAGE}">
```

Comment:
@leeyimin Thanks for your inputs. But what do you mean by missing? Action field? `adminEmailComposePage` is there right?